zebedee/client.go

```diff
@@ -469,6 +469,31 @@ func (c *Client) GetPublishedData(ctx context.Context, uriString string) ([]byte
 	return content, nil
 }
 
+// GetPublishedIndex returns PublishedIndex
+func (c *Client) GetPublishedIndex(ctx context.Context, params *PublishedIndexRequestParams) (PublishedIndex, error) {
+	reqURL := "/publishedindex"
```

Comment:
I notice in the functions above, they form reqURL using `c.createRequestURL` whereas this one is hard coded

Reply:
Other functions have dynamic parameters and can take a collectionId. This one only has a simple endpoint so this isn't currently needed and can simply be a string.